### PR TITLE
docs: add site url (fixes links)

### PR DESCRIPTION
**Pull request by hollandjg**
# Description

Fix links in documentation
# Type of change

*Delete as appropriate:*
- **docs**: Documentation only changes

### CODE REVIEW

File: mkdocs.yml [hunk 1]
```diff
@@ -1,6 +1,7 @@
 # yaml-language-server: $schema=https://squidfunk.github.io/mkdocs-material/schema.json
 
 site_name: Automated Research Assistant
+site_url: 'https://autoresearch.github.io/autora/'
```
Comment (younesStrittmatter) [inline mkdocs.yml:4]: site_url: 'https://autoresearch.github.io/'

**Review by younesStrittmatter — CHANGES_REQUESTED** [review]
the collab work (localy after build with the site_url without autora)

File: mkdocs.yml (hunk 1 above)
Comment (hollandjg) [reply mkdocs.yml:4]: I'm not sure that that's the right thing to do – it conflicts with the documentation on https://www.mkdocs.org/user-guide/configuration/#site_url, so I think if we do it, we risk breaking something else later. 
I've added a suggestion to fix the links using the other parameters available on the `page` object

**Review by younesStrittmatter — APPROVED** [review]
looks good to me

**Pull request merged**